utilities_common/platform_sfputil_helper.py

```diff
@@ -64,3 +66,17 @@ def get_asic_id_for_logical_port(port):
 def get_physical_to_logical():
 
     return platform_sfputil.physical_to_logical
+
+
+def get_interface_alias(port, db):
```

Comment:
This is better, and we can move forward with this for now, but eventually we should move this to another file in utilities_common and share that one function among _all_ `show` and `config` subcommands.